resources/web/wwi/nodes/WbWorld.js

```diff
@@ -20,6 +20,7 @@ export default class WbWorld {
   }
 
   static computeUpVector() {
-    WbWorld.instance.upVector = glm.vec3(WbWorld.instance.coordinateSystem[0] === 'U' ? 1 : 0, WbWorld.instance.coordinateSystem[1] === 'U' ? 1 : 0, WbWorld.instance.coordinateSystem[2] === 'U' ? 1 : 0);
+    WbWorld.instance.upVector = glm.vec3(WbWorld.instance.coordinateSystem[0] === 'U' ? 1 : 0,
+      WbWorld.instance.coordinateSystem[1] === 'U' ? 1 : 0, WbWorld.instance.coordinateSystem[2] === 'U' ? 1 : 0);
```

Comment:
Minor comment: this would be more readable if `WbWorld.instance.coordinateSystem` will be stored in a local variable.
So then it would even not be needed to split the line.

This seems what does the Javascript coding standard best practiced suggests.